src/components/user-profile/profile.jsx

```diff
@@ -7,39 +7,50 @@ function Profile() {
     const [businessName, setBusinessName] = useState('');
     const [bio, setBio] = useState('');
     const [error, setError] = useState('');
+    const navigate = useNavigate();
 
     const handleLogoChange = (event) => {
         setLogo(event.target.files[0]);
     };
 
     const handleSubmit = async (e) => {
         e.preventDefault();
-
-        if (!logo || !businessName || !bio) {
-            setError('Please fill in all fields');
+        const url = `${import.meta.env.VITE_BACKEND_ADDRESS}/api/create-profile`
+        const body = JSON.stringify({ businessName, bio });
+        if (!businessName || !bio) {
+            setError('Please fill in Name and Bio');
             return;
         }
```

Comment:
I think a better way to write this would be:
```
const errors = [];
if(!businessName){
   errors[] = "Name"
}
if(!bio){
   errors[] = "Bio"
}
if(errors.length > 0){
  const message = `Please fill in: ${errors.join(",")}`;
   setError(message);
  return;
}
```
This will make you message to the user much more clear. In case they filled out the name but not the bio.

Reply:
thank you for this suggestion. More specific error messages would definitely be more beneficial to the user